src/region/transformUtils.ts

```diff
@@ -0,0 +1,24 @@
+export type Point = {
+    x: number;
+    y: number;
+};
+
+export const invertYCoordinate = ({ x, y }: Point, height: number): Point => ({
+    x,
+    y: height > 0 ? height - y : y,
+});
+
+export const rotatePoint = ({ x, y }: Point, rotationInDegrees: number): Point => {
+    const { cos, sin } = Math;
```

Comment:
Not sure I'm a fan of destructuring static methods, though I don't have a strong reason. Can we use `Math.cos` inline?